app/src/main/java/fr/neamar/kiss/preference/ColorPreference.java

```diff
@@ -99,22 +102,33 @@ public void onGlobalLayout() {
         Button button1 = view.findViewById(R.id.colorTransparentDark);
         button1.setOnClickListener(new View.OnClickListener() {
             public void onClick(View v) {
-                ColorPreference.this.onColorSelected(0xAA000000);
+                ColorPreference.this.onColorSelected(COLOR_DARK_TRANSPARENT);
             }
         });
+        if(ColorPreference.this.selectedColor == COLOR_DARK_TRANSPARENT) {
+            button1.setTypeface(null, Typeface.BOLD);
```

Comment:
might be worth creating a function selectButton(View button) to set the typeface and the color?